Makefile

```diff
@@ -8,10 +8,6 @@ gen: solidity-wrappers
 
 SOLIDITY_DIR = .
 SOLIDITY_SRC_DIR = $(SOLIDITY_DIR)/src
-solidity-wrappers: $(SOLIDITY_SRC_DIR)/QuantumGravityBridge.sol
-	cd $(SOLIDITY_SRC_DIR) ; \
-	for file in $(^F) ; do \
-			mkdir -p ../wrappers/$${file} ; \
-			echo abigen --type=peggy --pkg wrappers --out=../wrappers/$${file}/wrapper.go --sol $${file} ; \
-			abigen --type=peggy --pkg wrappers --out=../wrappers/$${file}/wrapper.go --sol $${file} ; \
-	done
+CONTRACTS = QuantumGravityBridge.sol
+solidity-wrappers:
+	SOLIDITY_SRC_DIR=$(SOLIDITY_SRC_DIR) CONTRACTS=$(CONTRACTS) /bin/bash ./scripts/gen.sh
```

Comment:
For proper separation, can you make these into parameters instead of globals? As-is, it's exposing the implementation details of the name of these internal parameters in the script.

Reply:
be07c7e